src/frontend/screens/AppPasscode/NewPasscode/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { View } from "react-native";
+import { SplashScreen } from "./Splash";
+
+enum ScreenState {
+  splash,
```

Comment:
nit but maybe rename to something closer to `intro`? i think of the app/os splashscreen when i see splash